jrodriguezdiazz/jrodriguezdiazz-portfolio
# ✨ Add the Skills section

## Description



- Add a new section, **_Skills_** what I have

- Improve accessibility by adding an aria-label property for Github, LinkedIn, and Twitter links [`index.html`](https://bit.ly/3BaS8T2)

- Add CSS rule missing to focus the titles of each section, [`assets/css/reusable-css-classes.css`](https://bit.ly/2UawEoW)

- Delete CSS rule missing to center the title of the [about CSS file](https://bit.ly/3iepvM3)

---

## Motivation and Context





As a personal portfolio, I need to show what tools, programming languages, or skills I know.

---

## How Has This Been Tested?






```shell
git clone https://github.com/jrodriguezdiazz/jrodriguezdiazz-portfolio.git && git checkout feature-skills && git pull -r
```

- Open the index.html in your browser
- Inspect the document
- Change the size of the browser
- Check the feature of navigation on mobile devices, the maximum value of the screen must be 766px
- You should show the Skills section at the end of the document
- Click each option (_Front-End_, _Back-End_, and _Tools_) and check if when your click the arrow displays the list of skills

---

## Screenshots (if appropriate):

|                                                                                                                 |                                                                                                                 |
| :-------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------: |
| ![image](https://user-images.githubusercontent.com/24766287/125884321-e509e368-1a89-4e01-b7da-a0737f9b9c56.png) | ![image](https://user-images.githubusercontent.com/24766287/125884431-a4426191-93ea-42db-995f-10d7f5077b3e.png) |

---

## Types of Changes




---

## Checklist:





---

# Other Notes



🚧 For the moment the values of the skills are thus maintained to be changed in the future by real data
